test/src/test/java/hudson/model/AbstractProject2Test.java

```diff
@@ -56,16 +56,20 @@ public void upstreamDownstreamExportApi() throws Exception {
             grant(Item.READ).everywhere().to("alice").
             grant(Item.READ).onItems(us).to("bob").
             grant(Item.READ).onItems(ds).to("charlie"));
-        String api = r.createWebClient().login("alice").goTo(us.getUrl() + "api/json?pretty", null).getWebResponse().getContentAsString();
+
+        String api = r.createWebClient().withBasicCredentials("alice", "alice").goTo(us.getUrl() + "api/json?pretty", null).getWebResponse().getContentAsString();
```

Comment:
I thought you could use just `withBasicCredentials("alice")` if the password was the same, as it is for `DummySecurityRealm`?